packages/frint-react/src/components/Region.spec.js

```diff
@@ -133,6 +133,7 @@ describe('frint-react › components › Region', function () {
     );
 
     // verify
+    console.warn.restore(); // eslint-disable-line no-console
```

Comment:
Not sure why the method was already stubbed, do you @fahad19 have any clue on this?

Reply:
Nevermind, it was something related to my local env.